unleash/src/test/java/com/silvercar/unleash/integration/ClientSpecificationTest.java

```diff
@@ -96,11 +96,13 @@ private Unleash setupUnleash(TestDefinition testDefinition) throws URISyntaxExce
         // Required because the client is available before it may have had the chance to talk with the API
         String backupFile = writeUnleashBackup(testDefinition);
 
+        // TODO: Determine why fetch interval causes tests to fail when debugging.
```

Comment:
You mean running the tests in the debugger fails the test, but the test passes otherwise?

Reply:
Correct, the cached toggles get changed by the internal to a different test iteration